src/clientManager.js

```diff
@@ -20,6 +20,19 @@ ClientManager.prototype.findClient = function(name) {
 	return false;
 };
 
+ClientManager.prototype.writeUser = function(userDict) {
+	var users_path = Helper.HOME + "/users" ;
+	var user_path = users_path + "/" + userDict.user + ".json";
+
+	mkdirp(users_path);
+	fs.writeFileSync(
+		user_path,
+		JSON.stringify(userDict, null, "  ")
+	);
+	// No effect on Windows (and no easy way to do something equivalent)
+	fs.chmodSync(user_path, "0660");
```

Comment:
@xPaw @maxpoulin64 . I did it on purpose, thinking it was harmless and added extra bonus of fixing permissions from old versions, but your points are valid, I'll change.

TODO:

- [ ] do not reset permissions on every file edit